assets/static/css/data-discovery.css

```diff
@@ -241,3 +241,22 @@ input.ui-button {
 #expand-collapse.collapsed {
     left: -12px;
 }
+
+.teal-button
+{
+    background-image:none;
+    background-color:darkcyan;
```

Comment:
This color should be: `var(--color-headerMiddle)`
It's a bad name, but it's the right one.

If you don't this won't work in the other template.

Reply:
ok, works as suggested.